ui/components/EventMembers/index.tsx

```diff
@@ -114,11 +118,30 @@ const EventMembers = ({ collection, currentUser }) => {
       fetching: loading,
       error,
     },
+    searchApprovedMembers
   ] = useQuery({
     query: COLLECTION_MEMBERS_QUERY,
-    variables: { collectionId: collection.id, offset: 0, limit: 1000 },
+    variables: { collectionId: collection.id, usernameStartsWith: searchString, offset: 0, limit: 1000 },
+    pause: true,
   });
 
+  const debouncedSearchMembers = useMemo(() => {
+    return debounce(searchApprovedMembers, 300, { leading: true });
+  }, [searchApprovedMembers]);
+
+  const items = useMemo(() => {
+    if (loading || !approvedMembers) {
+      return [];
+    }
+    return approvedMembers
+  }, [approvedMembers, loading]);
+
+  useEffect(() => {
+    // if (tooShortSearch) return;
```

Comment:
don't we want this limitation?

Reply:
Yeah, I think one character makes sense when filtering a list where it's fine to have many results, but two characters are better for mentions - where you definitely never want to fill the autocomplete selector with all usernames starting with "a".